### PR TITLE
Shortened proofs 2

### PR DESCRIPTION
**Pull request by GinoGiotto**
Continuation of https://github.com/metamath/set.mm/pull/3127. Proof minimizer ran between the 1000th and the 2000th theorems that don't have a `proof modification is discouraged` label in their comment.

### CODE REVIEW

**Comment by GinoGiotto** [comment]
I just noticed that the theorem `exintr` and `alexan` are exactly the same, maybe one of them could be removed?

**Comment by wlammen** [comment]
I suggest to keep exintr, as this was introduced earlier, but with the shorter proof of alexan.

**Comment by GinoGiotto** [comment]
Ok, so no further edits from this pull request

**Comment by wlammen** [comment]
Except correcting the alexan/exintr duplicate I see no other issue with this pull request.

**Review by wlammen — APPROVED** [review]

File: set.mm [hunk 1]
```diff
@@ -15665,7 +15660,7 @@ only postulated (that is, axiomatic) rule of inference of predicate
   $( Introduce a conjunct in the scope of an existential quantifier.
      (Contributed by NM, 11-Aug-1993.) $)
```
Comment (benjub) [inline set.mm:15661]:
```suggestion
     (Contributed by NM, 11-Aug-1993.)  (Proof shortened by BJ,
     16-Sep-2022.) $)
```
Try not to remove credits, please.

Comment (GinoGiotto) [reply set.mm:15661]: I forgot, thanks

**Review by icecream17 — APPROVED** [review]

**Pull request merged**